extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.css

```diff
@@ -0,0 +1,17 @@
+.ve-ui-wikiaSourceModeDialog-content .oo-ui-window-body .oo-ui-textInputWidget {
+	height: 100%;
+	max-width: none;
+	padding: 0.75em;
+	width: 100%;
+}
+.ve-ui-wikiaSourceModeDialog-content .oo-ui-window-body .oo-ui-textInputWidget textarea {
+	font-family: monospace;
+	font-size: 13px;
+	height: 100%;
+	line-height: 140%;
+	width: 100%;
+}
+.ve-ui-wikiaSourceModeDialog-content .oo-ui-window-body .oo-ui-textInputWidget .ve-ui-wikiaSourceModeDialog-helplink {
+	font-weight: bold;
+	line-height: 3em;
```

Comment:
As it stands, we cannot currently load sass files; but once we can this file will be replaced with sass. Font-sizes, paddings, etc will then be variable driven.